docs/source/about.rst

```diff
@@ -22,10 +22,12 @@ grown into a vivid community of researchers,  users, and developers from
 around the world.
 
 
-.. image:: ../images/ASReviewLAB_explanation_Website_v3_wit.png
-    :width: 320
-    :align: center
-    :alt: ASReview LAB overview
+.. raw:: html
+
+    <div style="position: relative; margin-bottom: 2em; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+        <iframe src="https://www.youtube.com/embed/k-a2SCq-LtA" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
+    </div>
```

Comment:
This looks hacky and hard to maintain. It doesn't work for the PDF version. I think it is nicer to implement the solid-looking sphinx-youtube extension https://github.com/sphinx-contrib/youtube